Apromore-Custom-Plugins/Process-Simulation-Info-Logic/src/main/java/org/apromore/processsimulation/service/SimulationInfoService.java

```diff
@@ -204,14 +206,18 @@ private void deriveTimetable(
 
         CalendarModel calendarModel = simulationData.getCalendarModel();
 
-        if (calendarModel.getName().equals(SimulationData.DEFAULT_CALENDAR_NAME)) {
```

Comment:
Is this constant used anymore? If not its better if we remove it.

Reply:
It is using in SimulationInfoServiceTest class. So, I decied to keep it with changing name and remove the one constant that i have created in SimulationInfoService class